website/versioned_docs/version-1.0.2/compaction.md

```diff
@@ -13,7 +13,7 @@ not applicable to Copy On Write(COW) tables and only applies to MOR tables.
 
 ### Why MOR tables need compaction?
 To understand the significance of compaction in MOR tables, it is helpful to understand the MOR table layout first. In Hudi, 
-data is organized in terms of [file groups](https://hudi.apache.org/docs/file_layouts/). Each file group in a MOR table 
+data is organized in terms of [file groups](/docs/storage_layouts/). Each file group in a MOR table 
```

Comment:
this is the correct change. why are other relative path changes not following this?